server/views/pages/plan.njk

```diff
@@ -54,7 +54,7 @@
 }) %}
 
 {% block pageTitle %}
-    {% if not errors.domain %}
+    {% if not (errors.domain | length) %}
```

Comment:
Any reason we don't just check the size of 'errors' in general? 

Is it because `errors` is always shaped like
```
{
   query: {},
   body: {},
   params: {},
   domain: {}
}
```
?

Reply:
Yes, precisely. We can't check size or for null on the main `errors` object because it always has these fields.